docs/technology.md

```diff
@@ -43,3 +43,121 @@ The slowest step in the pipeline is the graph database writer.
 It takes large chunks of data, converts it all to queries and submits generally long running queries it has to await.
 While this is happening, python can suspend the task waiting on the database allowing the other steps to continue processing.
 This allows for the extractor and interpreter to continue processing data and not be constrained by the database writer.
+
+## Migrations 
```

Comment:
Not in the changes, but above `asyncronously` should be `asynchronously` and also
`exectuion`
and also
`amout`

Sorry I asked chatGPT to check the entire file :)

Reply:
Thank you!!